src/number_format.js

```diff
@@ -21,20 +21,23 @@ const propTypes = {
     PropTypes.number,
     PropTypes.string
   ]),
+  allowNegative: PropTypes.bool,
   customInput: PropTypes.func
 };
 
 const defaultProps = {
   displayType: 'input',
   decimalSeparator: '.',
-  decimalPrecision: false
+  decimalPrecision: false,
+  allowNegative: true
```

Comment:
I think default value should be true if format prop is not defined and should be false if format props defined.